Code/Editor/Plugins/ProjectSettingsTool/ProjectSettingsToolWindow.cpp

```diff
@@ -364,6 +364,12 @@ namespace ProjectSettingsTool
             }
         }
 
+        if (index == -1)
+        {
+            // If there is no current tab, return
+            return;
+        }
+
```

Comment:
Should this logic moved to the top of `ResizeTabs`, or do we want the tabs to be given the `Ignored` size policy when no tabs are selected?

Reply:
I think in the case that the index is 0 the `m_ui->platformTabs` container would have a size of 0